common/colour/reports/pre-a11y-2.json

```
@@ -0,0 +1,444 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/common/colour/colour-en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Evaluation of how various text appears on different backgrounds.",
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": ["#text-white", "#text-sm-left", "#text-sm-right", "#bg-primary", "#bg-success","#bg-info", "#bg-danger", "#bg-warning", "#text-muted", "#text-primary", "#text-success", "#text-info", "#text-danger", "#text-left", "#text-right", "#text-center", "#text-justify", "#text-nowrap", "#bg-cover", "#bg-center", "#bg-norepeat", "#bg-gctheme", "#bg-darker", "#bg-dark"]
+    }
+  },
+
+  "earl:assertedBy": {
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Brahim Mahadi Wachilli (Github: @BrahimMahadi)",
+      "foaf:homepage": "https://github.com/BrahimMahadi",
+      "@type": ["earl:Assertor", "foaf:Person"]
+    }
+  },
+
+  "dct:date": "2023-08-16",
+  "dct:description": "Analyzing and exploring the subject and produce a pre-evaluation of all WCAG 2.1 SC at level AA.",
+  "acr:involvesExpertise": [
+    "wbv:acr/topics#aria",
+    "Javascript"
+  ],
```

I don't think the tested subject require WAI-ARIA expertise neither JS expertise.

Leave it blank like:

```suggestion
  "acr:involvesExpertise": [],
```

---

Fixed in commit "Made requested changes"